templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -672,12 +669,12 @@ Resources:
         PublicSubnet2CIDR: !Ref 'PublicSubnet2CIDR'
         VPCCIDR: !Ref 'VPCCIDR'
 
-  JiraDCStack:
+  CrowdDCStack:
     DependsOn: VPCStack
     Type: AWS::CloudFormation::Stack
     Properties:
       TemplateURL: !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}templates/quickstart-jira-dc.template.yaml
+        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}templates/quickstart-crowd-dc.template.yaml
```

Comment:
Need to add the `quickstart-crowd-dc.template.yaml` template to the bucket `aws-quickstart`.

Reply:
@nanux @bpar476 do you know who I could reach out to for assistance on this?

Reply:
Tony will deploy the quick start once it is setup. In the meantime, I would use one of our S3 buckets to test this feature. You can use [Taskcat](https://github.com/aws-quickstart/taskcat) to upload the quick start and all of it submodules to the S3 bucket.

Reply:
Thanks, let me look into Taskcat.